src/main/resources/requests/searchDescriptionRequests.ts

```diff
@@ -1,11 +1,15 @@
-import {dataManagementApi} from './utils/axiosConfig';
+//import {dataManagementApi} from './utils/axiosConfig';
+import fs from 'fs';
+import path from 'path';
 
 export class SearchDescriptionRequests {
 
   public async getStatusDescriptionList(): Promise<Array<any>> {
     try {
-      const response = await dataManagementApi.get('/courteventglossary');
-      return response.data;
+      //const response = await dataManagementApi.get('/courteventglossary');
+      const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/StatusDescription.json'), 'utf-8');
```

Comment:
dont forget to remove the json file too when you switch this back

Reply:
sure